### ISSUE TITLE
PCA 正規化に関する質問

### ISSUE THREAD

**Issue by shitian-ni** [Koteraken/Experiment2017#8]
[PCA2017.py](https://github.com/Koteraken/Experiment2017/blob/master/PCA2017.py)に行列の正規化`dfs = df.apply(lambda x: (x-x.mean())/x.std(), axis=0).fillna(0)`に関して、
sklearnのPCAは分散を使い、分散が大きい方を主成分に多く入れるはずだが、正規化によって全部の要素の分散が１になるので一番大きい分散や一番小さい分散がないと思います。
> [元のデータをなるべく反映させるのならば共分散を，単位を揃えて標準化する必要があるのならば相関](https://oshiete.goo.ne.jp/qa/531412.html)

相関行列を使うべきかと思いますが、[StandardScaler](https://github.com/scikit-learn/scikit-learn/issues/2689)を使うらしいです。
PCA2017.pyの行列の正規化で分散を全部1になったらどうなるのかを解説していただけますか。
